### PR TITLE
Small fixes in phantom theme

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
Some small fixes for the Phantom (Umbraco v9+)

### CODE REVIEW

File: src/Articulate/App_Plugins/Articulate/Themes/Phantom/Views/Master.cshtml [hunk 1]
```diff
@@ -20,25 +20,24 @@
     <meta name="MobileOptimized" content="320" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
-    @{ Html
-            .RequiresJs("https://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js")
-            .RequiresThemedCssFolder(Model);
+    @{ Smidge.RequiresJs("https://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js")
```
Comment (Shazwazza) [inline src/Articulate/App_Plugins/Articulate/Themes/Phantom/Views/Master.cshtml]: IIRC the preferred way to do this is to use bundles and render them that way. For example, this is in the vapor theme https://github.com/Shazwazza/Articulate/blob/release/5.0.0/src/Articulate/App_Plugins/Articulate/Themes/VAPOR/Views/Master.cshtml#L25

File: src/Articulate/App_Plugins/Articulate/Themes/Phantom/Views/Master.cshtml [hunk 2]
```diff
@@ -20,25 +20,24 @@
...
 
-    <div class="content">
-        @RenderBody()
-    </div>
+<div class="content">
+    @RenderBody()
+</div>
     
-    @Html.RenderJsHere()
+@await Smidge.JsHereAsync()
```
Comment (Shazwazza) [inline src/Articulate/App_Plugins/Articulate/Themes/Phantom/Views/Master.cshtml]: Similarly here, I think bundles are preferred for example https://github.com/Shazwazza/Articulate/blob/release/5.0.0/src/Articulate/App_Plugins/Articulate/Themes/VAPOR/Views/Master.cshtml#L68

**Review by Shazwazza — CHANGES_REQUESTED** [review]

File: src/Articulate/App_Plugins/Articulate/Themes/Phantom/Views/Master.cshtml (hunk 1 above)
Comment (erikjanwestendorp) [reply src/Articulate/App_Plugins/Articulate/Themes/Phantom/Views/Master.cshtml]: That's great, didn't know that. 🙈

File: src/Articulate/App_Plugins/Articulate/Themes/Phantom/Views/Master.cshtml (hunk 2 above)
Comment (erikjanwestendorp) [reply src/Articulate/App_Plugins/Articulate/Themes/Phantom/Views/Master.cshtml]: Updated 😄

**Comment by erikjanwestendorp** [comment]
@Shazwazza Thanks for the review, just updated the PR with the desired changes.

**Review by Shazwazza — APPROVED** [review]
I think it looks good! I trust that you've tested, etc... and it all works :)

**Pull request merged**